contrib/guix/manifest.scm

```diff
@@ -517,7 +517,6 @@ inspecting signatures in Mach-O binaries.")
         moreutils
         ;; Compression and archiving
         tar
-        bzip2
```

Comment:
29895: should drop it from Dockerfile-s too I guess

Reply:
I'm not using `contrib/guix/Dockerfile` either. I guess we could point `INSTALL.md` to `contrib/containers/guix/Dockerfile` and drop `contrib/guix/Dockerfile` then. @knst @PastaPastaPasta ?